stubs/fpdf2/fpdf/graphics_state.pyi

```diff
@@ -52,17 +52,17 @@ class GraphicsStateMixin:
     @current_font.setter
     def current_font(self, v: dict[str, Any]) -> None: ...
     @property
-    def dash_pattern(self): ...
+    def dash_pattern(self) -> dict[str, float]: ...
     @dash_pattern.setter
-    def dash_pattern(self, v) -> None: ...
+    def dash_pattern(self, v: dict[str, float]) -> None: ...
     @property
-    def line_width(self): ...
+    def line_width(self) -> float: ...
     @line_width.setter
-    def line_width(self, v) -> None: ...
+    def line_width(self, v: float) -> None: ...
     @property
-    def text_mode(self): ...
+    def text_mode(self) -> int: ...
```

Comment:
(You'll need to add `from .enums import TextMode` at the top of the file):

```suggestion
    def text_mode(self) -> TextMode: ...
```

It's set to `TextMode.FILL` here and, because it's an enum, `TextMode.FILL` is an instance of `TextMode`: https://github.com/PyFPDF/fpdf2/blob/dcb87ea11709f40effbe26fe3746e48735b6f708/fpdf/fpdf.py#L330

The setter method in `graphics_state.py` also auto-converts things to `TextMode` objects when the attribute is set: https://github.com/PyFPDF/fpdf2/blob/dcb87ea11709f40effbe26fe3746e48735b6f708/fpdf/graphics_state.py#L169

Reply:
> from .enums import TextMode

i did not receive email if your feedback. Is something wrong here?